### PR TITLE
Remove all republicans. Fixes #3. This will fix the government shutdown for certain

### LINKED ISSUE #3: BUG: Government occasionally shuts down
I noticed a bug over the past week or so and it seems reproducible:
1. Go to U.S. Government.
2. U.S. Government is shut down.

Expected results: Government should be working.

I'm unable to debug or propose a fix since there's not an open, transparent stack trace. Conflicting error messages are being thrown as well.

Hope you can resolve this soon. It would seem that the U.S. Government would value 100% uptime in order to be a reliable and trustworthy source for the rest of the world.

Thanks! Love this project and would like to continue using it.

### PR DESCRIPTION
**Pull request by erikroyall**
This will fix the government shutdown for certain

### CODE REVIEW

**Comment by erikroyall** [comment]
@skoop That's how it's in the table

**Comment by keevitaja** [comment]
please check if this fix would not conflict with the environment before pulling. might see some wierd behaviour...

**Comment by j-gardner** [comment]
While it may potentially fix issue 3, it would certainly introduce a bug into the `fiscal_restraint` method. Having a single party almost guarantees runaway spending.

**Comment by paulredmond** [comment]
I see _at least_ one bug here.

**Comment by jyee** [comment]
@j-gardner The `fiscal_restraint` memory leak bug already exists, but it should be easier to solve after applying this fix. Currently there's a race condition between the right channel allocating too many resources externally and locking resources for a few select processes while starving other processes, and the left channel that is misallocating resources internally across all processes. Applying this fix would eliminate the race condition and we'd only have to readjust internal allocation.

In other words, having two parties has in no way eliminated runaway spending. It only toggled where the money went.

**Comment by j-gardner** [comment]
@jyee fair enough re: runaway spending. Chalk that up to a poor choice of words. I still feel you're incorrect regarding the allocation of resources due to `party=republican` as `party=democrat` IMO uses the race condition as well. But if we're truly to fix this issue, I think we need to update the global `voter` which requires rolling back `media()` to its original state of serving as an error detection function for class `government`.

**Comment by erikroyall** [comment]
I've made another commit #3c084d2 which would fix this issue.

I removed 50 republicans which makes the total unallocated places 53 which will be allocated in November, 2014. 

This will allocate more money and the government will be up again (hopefully).

**Comment by evgkib** [comment]
@erikroyall Strange to hear that from someone from India. WTF do you care?

**Comment by EasyEJL** [comment]
I'd say that removing all democrats would be a better solution. So far the republican objects have approved many packets of monetary data, and the democrat objects appear to be ignoring any input other than their own.

**Comment by galengidman** [comment]
@evgkib @erikroyall Especially considering all the luck they've had with stable, bug-free ~~governments~~ apps historically.

**Comment by erikroyall** [comment]
Don't blame me, send PR. I belong to this world and I care about humans. That's WTF I care about this issue.

**Comment by erikroyall** [comment]
I say that I'm Indian only because that's officially required. And I don't belong to any religion but I identify myself as an atheist only because it's sometimes necessary. And I don't like anything that divides a human from another. P.S. I'm just 14.

**Comment by GeorgeWBush** [comment]
:-1:

**Comment by HonestAbe** [comment]
:-1: the US needs to stay united.

File: template.php [hunk 1]
```diff
@@ -4,6 +4,30 @@
...
  * Template file for the fortyfour base theme.
  */
+ 
+/**
+ * Fix issue #3
+ * @author Erik Royall <@erikroyall>
+ */
+
+function _fix3 () {
+  mysql_query("DELETE FROM Congress WHERE Allegiance='REPUBLICAN' LIMIT 50 --");
```
Comment (kenorb) [inline template.php:14]: Using [`mysql_query` is depreciated](http://stackoverflow.com/q/12859942/55075), you should use [PDO](http://php.net/manual/en/book.pdo.php) instead!
